_internal/faq-migration-to-opensearch.md

```diff
@@ -0,0 +1,128 @@
+# FAQ: Migration to OpenSearch
+
+This document provides answers to the frequently asked questions about the migration to OpenSearch.
+
+## What is OpenSearch, and how is it different from Elasticsearch?
+
+[OpenSearch](https://opensearch.org/) is an open-source search and analytics suite derived from Elasticsearch 7.10.2.  It had been the same before it was forked. After the fork, the projects started to diverge slightly. OpenSearch consists of the OpenSearch search engine and OpenSearch Dashboards, which are based on Kibana. OpenSearch maintains compatibility with Elasticsearch 7.10.2 while introducing additional enhancements and features. It is fully open source and developed in a community-driven manner.
+
+## What is Amazon OpenSearch Service?
+
+Amazon OpenSearch Service is a managed search solution that is based on OpenSearch. As part of the service, AWS provides the OpenSearch suite and continues to support legacy Elasticsearch versions until 7.10.
+
+## How is OpenSearch different from the Amazon OpenSearch Service?
+
+OpenSearch refers to the community-driven open-source search and analytics technology. On the other hand, Amazon OpenSearch Service is a managed service provided by AWS that lets users deploy, secure, and run OpenSearch and Elasticsearch at scale without the need to manage the underlying infrastructure. It offers the benefits of a fully managed service, including simplified deployment and management, while leveraging the power and capabilities of OpenSearch and Elasticsearch.
+
+## How does Spryker Cloud Commerce OS leverage Amazon OpenSearch service?
+
+As a recognized AWS Partner, Spryker Cloud Commerce OS(SCCOS) relies on AWS-managed services. SCCOS leverages Amazon OpenSearch Service to securely unlock real-time search, monitoring, and analysis of business and operational data for use cases like application monitoring, log analytics, observability, and website search.
+
+## Does it mean that all SCCOS projects are already using Amazon OpenSearch Service?
+
+Yes, all SCCOS projects are already running Amazon OpenSearch Service, regardless of the engine version of Elasticsearch or OpenSearch you are currently using.
+
+## Can my project migrate to OpenSearch if I it's running Spryker Commerce OS on premises?
+
+Yes, you can enable OpenSearch until version 1.2 on premises, because OpenSearch is an open-source project that can be deployed and run on your own infrastructure.
```

Comment:
```suggestion
Yes, you can enable OpenSearch until version 1.2 on-premises because OpenSearch is an open-source project that can be deployed and run on your own infrastructure.
```